test/Microsoft.AspNet.Razor.Test/Utils/TestFile.cs

```diff
@@ -27,7 +27,9 @@ public static TestFile Create(string localResourceName)
 
         public Stream OpenRead()
         {
-            var strm = Assembly.GetManifestResourceStream(ResourceName);
+            string resourceName = Assembly.GetName().Name + "." + ResourceName.Replace('/', '.');
+
+            var strm = Assembly.GetManifestResourceStream(resourceName);
             if (strm == null)
             {
                 Assert.True(false, string.Format("Manifest resource: {0} not found", ResourceName));
```

Comment:
`ResourceName` is no longer valid here because it's not pointing to the same location as requested.